config/locales/server.en.yml

```diff
@@ -1766,6 +1766,7 @@ en:
     search_query_log_max_retention_days: "Maximum amount of time to keep search queries, in days."
     search_ignore_accents: "Accent marks in search queries are disregarded if this setting is enabled, allowing users to find results even if they don't input the correct accents."
     search_default_sort_order: "Default sort order for full-page search"
+    search_experience: "The default position and appearance of search on desktop devices."
```

Comment:
If there is only one sentence please leave off the period at the end, see https://meta.discourse.org/t/formatting-text-in-discourse-documentation-and-uis/324637